modules/ROOT/edits/index.adoc

```diff
@@ -0,0 +1,64 @@
+= Anypoint Runtime Fabric Overview
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Anypoint Runtime Fabric is a container service that allows you to run Mule applications, composite APIs and API gateways across any environment with centralized management via Anypoint Platform.
+
+Some of the capabilities of Anypoint Runtime Fabric include:
+
+* Isolation between applications by running a separate Mule runtime server per application.
+* Ability to run multiple versions of Mule runtime server on the same set of resources.
+* Scaling applications across multiple replicas.
+* Automated application fail-over.
+* Application management with Anypoint Runtime Manager.
+
+[NOTE]
+Contact your Customer Success Manager for information on how enable Anypoint Runtime Fabric on your account.
+
+There are two management options for Anypoint Runtime Fabric:
+
```

Comment:
Because we do not use boldface to provide narrative headings (or for list items), I suggest reformatting these as simple bullet list items, including the linespace between list item and description.
Another alternative is to make these subtopics and add more content specific to each.